src/test/java/edu/wisc/my/messages/data/MessagesDataIT.java

```diff
@@ -0,0 +1,42 @@
+package edu.wisc.my.messages.data;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.springframework.core.io.DefaultResourceLoader;
+import org.springframework.core.io.ResourceLoader;
+import org.springframework.mock.env.MockEnvironment;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class MessagesDataIT {
+
+    /**
+   * Test that the data in the incoming text file is actually valid.
+   */
+  @Test
+  public void dataIsValid() {
+    
+    final Logger logger = LoggerFactory.getLogger(getClass());
+
+    MockEnvironment mockEnv = new MockEnvironment();
+    
+    // ToDo: It would be very cool is this value could automaticall stay in sync
```

Comment:
I think this automation is possible by doing this testing in a richer Springified testing context rather than in the bare unit test.

cf. https://github.com/UW-Madison-DoIT/uportal-messaging/blob/00715678320c226cf6665a7956271780a07530c3/src/test/java/edu/wisc/my/messages/controller/MessagesControllerTest.java#L41